NEWS

```diff
@@ -1,3 +1,36 @@
+Overview of changes leading to 2.7.0
+Thursday, July 9, 2020
+====================================
+- Use an implementation for round that always rounds up, some minor fluctuations
+  are expected on var font specially when hb-ot callback is used.
+- Fix an AAT's `kerx` issue on broken rendering of Devanagari Sangam MN.
+- Remove AAT's `lcar` table support from _get_ligature_carets API, not even much
+  use on macOS installed fonts (only two files).  GDEF support is the recommended
+  one and expected to work properly after issues fixed two releases ago.
+- Minor memory fixes to handle OOM better specially in hb-ft.
+- Minor .so files versioning scheme change and remove stable/unstable scheme
+  differences, was never used in practice (always default to stable scheme).
+- *Migration to Meson build system for building the package*
+  As Arch Linux, Alpine Linux and msys2 have migrated to the meson port, we have
+  become confident that the meson port is probably usable for packaging
+  the library -- but of course feedback is still welcomed and some minor issues
+  are expected.
+  As meson distribution tarballs are similar to source checkouts,
+  our recommend URL scheme for downloading the source tarball now is
+  https://github.com/harfbuzz/harfbuzz/archive/$pkgver.tar.gz
+  or using commit hash of the release like,
+  git+https://github.com/harfbuzz/harfbuzz#commit=$_commit (used by Arch Linux)
+  which is much more safe and reliable in terms of knowing which
+  exact changes has led to the release.
+  If you are just integrating the library into your software and don't have the
+  cycles to migrate to meson for now, there is also amalgam/unity support,
+  src/harfbuzz.cc. This is a single source file that includes the whole library,
+  provided along the source. However using it requires a minor study of what the
+  build configs (meson or autotools) do with flags and definitions that they pass
+  to the compiler -- thus we recommend use of the regular build system for most
+  cases and specially recommend against the use of the amalgam for *nix packages.
```

Comment:
I don’t kmow, but this entry feels to much for NEWS files. It should be shorter, a line or two with a link to documentation (wiki) page with more details. Something like:
```
- Migration to meson is progressing, and now is good time to try it and report any issues.
- Source URL changed, please use the new URL: XXX
  For more information, see XXX
```

Reply:
I kind of agree. How and when to use (and not use) the alternative library build method is probably better placed in docs somewhere with just a mention of the priority shift in NEWS.

Reply:
Great idea! Moved most of it to https://github.com/harfbuzz/harfbuzz/wiki/Notes-on-migration-to-meson feel free to edit that (or even rename the page)